src/components/SignTransactionWallets/Web3.tsx

```diff
@@ -107,61 +115,110 @@ export default function SignTransactionWeb3({
-        })}
-      </div>
-      <div className="SignTransactionWeb3-instructions">
-        {translate('SIGN_TX_WEB3_PROMPT', {
-          $walletName: walletConfig.name || WALLETS_CONFIG.WEB3.name
-        })}
-      </div>
-      <div className="SignTransactionWeb3-img">
-        <img width="150px" src={walletConfig.icon} />
-      </div>
-
-      <div className="SignTransactionWeb3-input">
-        <div className="SignTransactionWeb3-errors">
-          {walletState === WalletSigningState.REJECTED && (
-            <div className="SignTransactionWeb3-rejection">
-              {translate('SIGN_TX_WEB3_REJECTED')}
-            </div>
-          )}
-          {walletState === WalletSigningState.NETWORK_MISMATCH && (
-            <div className="SignTransactionWeb3-wrong-network">
-              {translate('SIGN_TX_WEB3_FAILED_NETWORK', {
-                $walletName: walletConfig.name,
-                $networkName: networkName
-              })}
-            </div>
-          )}
-          {walletState === WalletSigningState.ADDRESS_MISMATCH && (
-            <div className="SignTransactionWeb3-wrong-address">
-              {translate('SIGN_TX_WEB3_FAILED_ACCOUNT', {
-                $walletName: walletConfig.name,
-                $address: senderAccount.address
-              })}
-            </div>
-          )}
-          {walletState === WalletSigningState.SUBMITTING && (
-            <div className="SignTransactionWeb3-submitting">
-              {translate('SIGN_TX_SUBMITTING_PENDING')}
-            </div>
-          )}
-        </div>
-        <div className="SignTransactionWeb3-description">{translateRaw('SIGN_TX_EXPLANATION')}</div>
-        <div className="SignTransactionWeb3-footer">
-          <BusyBottom
-            type={
-              walletConfig.id === WalletId.METAMASK
-                ? BusyBottomConfig.METAMASK_SIGN
-                : BusyBottomConfig.GENERIC_WEB3
-            }
-          />
-        </div>
-      </div>
-    </>
+    <SignTransactionWeb3UI
+      walletConfig={walletConfig}
+      walletState={walletState}
+      networkName={networkName}
+      senderAccount={senderAccount}
+    />
   );
 }
+
+const Footer = styled.div`
+  width: 100%;
+  margin-top: 2em;
+`;
+
+const Web3ImgContainer = styled.div`
+  padding-bottom: 3em;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+
+  @media (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    padding-bottom: 1em;
+  }
+`;
+
+const Web3Img = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 150px;
+  & img {
+    width: 150px;
+  }
+`;
+
+export interface UIProps {
+  walletConfig: IWalletConfig;
+  walletState: WalletSigningState;
+  networkName: string;
+  senderAccount: StoreAccount;
+}
+
+export const SignTransactionWeb3UI = ({
+  walletConfig,
+  walletState,
+  networkName,
+  senderAccount
+}: UIProps) => (
+  <Box p="2.5em">
```

Comment:
I don't think we need padding at the bottom as there is already padding from an outer box.

Reply:
Also for the sides, it looks squished compared to `master`.

Reply:
This has not been resolved. I think the padding can be removed entirely as the outer `section` has the needed padding already.